js/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "omi-summer-lab",
+  "version": "1.0.0",
+  "description": "none",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/IntelLedger/omi-summer-lab.git"
+  },
+  "main": "lib/index.js",
+  "scripts": {
+    "compile-protobuf": "node ./scripts/compile_protobuf.js > ./lib/protobuf/protobuf_bundle.json",
+    "test": "standard && mocha --recursive spec --compilers js:babel-register -r babel-polyfill",
+    "watch:test": "mocha -w --recursive spec --compilers js:babel-register -r babel-polyfill",
+    "postinstall": "./scripts/install_sawtooth_sdk.sh && npm run compile-protobuf",
```

Comment:
/project is on mapped volume into a docker image from a linux host. Sawtooth development is untennable on windows. 

The node environment can be reproduces as follows:
$ docker create -t --rm -v /home/cian/Workspaces/sawtooth/omi-summer-lab:/project --name omi-node node:6-wheezy tail -f /dev/null
$ docker exec -i -t omi-node bash
docker $ cd /project/js
docker $ npm install
Produces the following warning (error) in the output:

npm info linkStuff omi-summer-lab@1.0.0
npm info lifecycle omi-summer-lab@1.0.0~install: omi-summer-lab@1.0.0
npm info lifecycle omi-summer-lab@1.0.0~postinstall: omi-summer-lab@1.0.0
npm WARN lifecycle omi-summer-lab@1.0.0~postinstall: cannot run in wd %s %s (wd=%s) omi-summer-lab@1.0.0 ./scripts/install_sawtooth_sdk.sh && npm run compile-protobuf /project/js
npm info lifecycle omi-summer-lab@1.0.0~prepublish: omi-summer-lab@1.0.0
omi-summer-lab@1.0.0 /project/js